QuickStart.txt

```diff
@@ -7,18 +7,20 @@ Running the TCK from the source
 1. Checkout TCK workspace in TCKHOME (e.g., "mail-tck").
      git clone git@github.com:eclipse-ee4j/mail-tck.git
    Change directory to $TCKHOME.
-2. Place jakarta.activation.jar (if using Java SE 9 or later)
-   and jakarta.mail.jar in any location. Let's call it JARPATH. 
+2. Place jakarta.activation-api.jar, angus-activation.jar
```

Comment:
_and the implementation of Jakarta Activation (for example angus-activation)_ instead of just _angus-activation.jar_

Reply:
Done